lib/services/project-service.ts

```diff
@@ -103,6 +105,17 @@ export class ProjectService implements IProjectService {
 			projectName,
 		});
 
+		try {
+			await this.$childProcess.exec(`git init ${projectDir}`);
+			await this.$childProcess.exec(`git -C ${projectDir} add --all`);
+			await this.$childProcess.exec(`git -C ${projectDir} commit -m "init"`);
```

Comment:
```suggestion
			await this.$childProcess.exec(`git -C ${projectDir} commit --no-verify -m "init"`);
```
Let's make sure the commit doesn't run pre-commit hooks (coming from an npm package for example) - and potentially fail here.